lib/components/onBoarding/subWidgets/energyLevelDescriptionWidget.dart

```diff
@@ -30,15 +28,15 @@ class EnergyLevelDescriptionWidget extends StatelessWidget {
                 items: [
                   DropdownMenuItem(
                     child: Text(AppLocalizations.of(context)!.morningPerson),
-                    value: AppLocalizations.of(context)!.morning,
+                    value:"Morning",
```

Comment:
Can you use the internationalized version of these strings?

Reply:
I don't think that there  is a need to internationalize values of DropdownMenuItem since they are identifiers, not displayed text.